example/ExamplePage.js

```diff
@@ -36,6 +36,10 @@ var ExamplePage = React.createClass({
     getInitialState: function() {
         return {
             tests: [
+                { label: "provided for test 1", value: 'data/Token.json' },
+                { label: "provided for test 2", value: 'data/TokenListing.json' },
+                { label: "provided for test 3", value: 'data/TokenProject.json' },
+                { label: "provided for test 4", value: 'data/TokenProjectLegalEntities.json' },
```

Comment:
We don't want these in the PR that we submit to networknt/react-schema-form because they are pretty specific to our application.  Could you break this commit into 2 or more

Reply:
Ok, I'll remove it as soon as possible